src/main/java/edu/ucsb/cs156/spring/backenddemo/controllers/ZipCodeController.java

```diff
@@ -18,7 +18,7 @@
 import io.swagger.v3.oas.annotations.Parameter;
 import io.swagger.v3.oas.annotations.tags.Tag;
 
-@Tag(name="Country Code info from http://api.zippopotam.us/us/{zipcode}")
+@Tag(name="Zip Code info from http://api.zippopotam.us/us/{zipcode}")
```

Comment:
Can we change this link to just http://api.zippopotam.us/ to make it consistent with the example site